### DISCUSSION TITLE
Links for the pinned repos in profile pages are not fully clickable

### DISCUSSION THREAD

**Discussion by lewhunt** [community/community#113546] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

I have spotted an issue with the pinned repo links on the profile pages, which may result in some people not being able to access the pinned repos, or finding it difficult to select.

When you attempt to select/click a pinned repo link in your profile, the link is only working when your cursor is positioned slightly higher up on the text link, giving a very small area of interactivity, see video below:

https://github.com/community/community/assets/9886284/c44475f0-161d-4b38-a0b0-4fbf92b502b2

**Comment by codylerum** [comment]
Yeah there is oddly a popover on that showing the same value as the link

**Comment by lewhunt** [comment]
This now seems to be fixed
